### ISSUE TITLE
Pathsep means that tox.ini needs a different command on windows and posix

### ISSUE THREAD

**Issue by djpugh** [PyCQA/prospector#296]
From #246 

Is there a reason about the choice of pathsep at https://github.com/PyCQA/prospector/blob/master/prospector/config/datatype.py#L1

On windows this is ";", but on POSIX it is ':' so using e.g. a tox config to build locally on windows and on a linux build server means that the command needs to be different - is it possible to change this, or also just use ':' to split (happy to create a PR for this) - I appreciate the colon would feature on windows for an absolute path, but it feels like this could be handled using a try/except

**Comment by carlio** [comment]
Hmm, looking at this code I think it'd make more sense to just add a new '--output-file' argument and stop trying to squash both format and target into the same setting.

**Comment by carlio** [comment]
Added that change into the develop branch, it'll be released as part of #295 once that is ready.

**Comment by carlio** [comment]
Reopneing to get into the habit of labelling issues fixed but not released :-)

**Comment by djpugh** [comment]
@carlio  Appreciate you've already merged this in #297 and #295, but I'm not sure the fix covers all edge cases (especially with absolute paths)

**Comment by carlio** [comment]
What other edge cases do you think there'll be? Certainly a PR with some test cases would be good! Feel free to reopen this or create a new issue

**Comment by djpugh** [comment]
Sure, will do - I'll create a follow on issue and some test cases
